src/Comments.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import FacebookProvider, {Like, Share} from 'react-facebook';

```

Comment:
@MrNice42 To fix broken build, use relative imports instead importing from 'react-facebook'.

```js
import FacebookProvider from './FacebookProvider';
import Like from './Like';
import Share from './Share';
```